pkg/mimir/modules.go

```diff
@@ -548,12 +548,9 @@ func (t *Mimir) initQueryFrontend() (serv services.Service, err error) {
 }
 
 func (t *Mimir) initRulerStorage() (serv services.Service, err error) {
-	// if the ruler is not configured and we're in single binary then let's just log an error and continue.
-	// unfortunately there is no way to generate a "default" config and compare default against actual
-	// to determine if it's unconfigured.  the following check, however, correctly tests this.
-	// Single binary integration tests will break if this ever drifts
+	// If the ruler is not configured and Mimir is running in monolithic mode, then we just skip starting the ruler.
 	if t.Cfg.isModuleEnabled(All) && t.Cfg.RulerStorage.IsDefaults() {
-		level.Info(util_log.Logger).Log("msg", "Ruler storage is not configured in single binary mode and will not be started.")
+		level.Info(util_log.Logger).Log("msg", "Ruler is not being started because the ruler storage has not been configured.")
```

Comment:
```suggestion
		level.Info(util_log.Logger).Log("msg", "The ruler is not being started because you need to configure the ruler storage.")
```
Remove ambiguous passive voice. (I assume a human needs to perform the configuration.) Vet for technical accuracy.